src/main/java/org/skriptlang/skript/lang/script/Script.java

```diff
@@ -219,4 +219,9 @@ public <T extends ScriptEvent> Set<T> getEvents(Class<T> type) {
 		);
 	}
 
+	@Override
+	public String toString() {
+		return this.getConfig().getFileName();
```

Comment:
This should return something more like the IDE's default suggestion, e.g. 
```java
"Script[" +
	"config=" + config +
	", structures=" + structures +
	", suppressedWarnings=" + suppressedWarnings +
	", scriptData=" + scriptData +
	", eventHandlers=" + eventHandlers +
	']';
```

Script objects are holders for the structures and the script data, they aren't analogous to files. They also might not have a properly-linked config with a useful path if they were created by some addon at runtime. The Config toString should contain the path name, not the Script object.